src/cms/views/error_handler/error_handler.py

```diff
@@ -22,6 +22,15 @@ def render_error_template(context):
     :return: The rendered template response
     :rtype: ~django.template.response.TemplateResponse
     """
+    # django-webpack-loader assumes request object in context
+    # and that this request object has a specific attribute set
+    context.update(
+        {
+            "request": type(
+                "dummy_request", (object,), {"_webpack_loader_used_tags": set()}
+            )()
+        }
+    )
```

Comment:
I think in this case we should just use the real request of the `handler400()`, `handler403()` etc...